test/dynamo/test_aot_autograd_cache.py

```diff
@@ -284,11 +284,107 @@ def fn(a, b):
         self.assertEqual(counters["aot_autograd"]["autograd_cache_hit"], 1)
         self.assertEqual(counters["aot_autograd"]["autograd_cache_saved"], 1)
 
+    @largeTensorTest("64GB", device=GPU_TYPE)
+    @parametrize("device", (GPU_TYPE,))
+    @parametrize("dtype", (torch.float16, torch.bfloat16))
+    @inductor_config.patch("fx_graph_cache", True)
+    @inductor_config.patch("fx_graph_remote_cache", False)
```

Comment:
just wondering: do you imagine that the general pattern for our caching unit tests will be to turn always turn off the remote cache (or if not, is there an intention behind which tests get the remote cache turned on)? I guess always turning it off in unit tests will make our internal "don't let sandcastle bloat the remote cache" situation better, although it means less local testing for the remote cache